### PR TITLE
get configs from service

### PR DESCRIPTION
**Pull request by yaronpx**

### CODE REVIEW

File: t/007-decryptCookieV3.t [hunk 1]
```diff
@@ -74,8 +74,8 @@ Process a valid V1 cookie
     	    pxconfig.cookie_secret = "perimeterx"
     	    pxconfig.px_debug = true
     	    pxconfig.block_enabled = true
+    	    pxconfig.send_page_requested_activity = false
```
Comment (nitzanpx) [inline t/007-decryptCookieV3.t]: on all modules, we should align that sending activities will be set to true by defaults (block and page_requested)

File: lib/px/utils/config_loader.lua [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    local config = require(config_file)
+    local ngx_timer_at = ngx.timer.at
+    local px_logger = require("px.utils.pxlogger").load(config_file)
+    -- set interval
+    local function load_on_timer()
+        local ok, err = ngx_timer_at(config.load_intreval, load_on_timer)
+        if not ok then
+            px_logger.error("Failed to schedule submit timer: " .. err)
+        end
+        _M.get_configuration(config_file)
```
Comment (nitzanpx) [inline lib/px/utils/config_loader.lua]: if the code above  
```  
if not ok then
            px_logger.error("Failed to schedule submit timer: " .. err)
            end
```
is triggered, why do you execute `_M.get_configuration(config_file)` ?

File: lib/px/utils/config_loader.lua [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+function _M.get_configuration(config_file)
+    local http = require "resty.http"
+    local config = require(config_file)
+    local px_logger = require("px.utils.pxlogger").load(config_file)
+    px_logger.debug("Fetching configuration")
+    local cjson = require "cjson"
+    local px_server = config.configuration_server
+    local px_port = config.configuration_server_port
+    local path = '/module'
```
Comment (nitzanpx) [inline lib/px/utils/config_loader.lua]: can  we put this in pxconstants aswell?

Comment (yaronpx) [inline lib/px/utils/config_loader.lua]: not expecting it to change, and it's more an internal thing.
but if you feel strongly about it, I don't really mind.

**Pull request merged**